messages/index.js

```diff
@@ -54,6 +55,8 @@ module.exports = {
       ru: "{{? n>=2 && n<=4 }}а{{?? n!=1 }}ов{{?}}",
       sk: "{{? n==1 }}ok{{??}}kov{{?}}",
       sv: "{{? n!=1 }}er{{?}}",
+      uk: "{{? n>=2 && n<=4 }}и{{?? n!=1 }}ів{{?}}"
+
```

Comment:
@switchy, here you add additional empty line, try remove it. Below, there are also some extra spaces at the beginning of the lines in some places. Try deleting them too. Maybe it will allow you to pass the tests.

Reply:
It did not help.

Reply:
My comment is not only talking about empty lines, but also about extra spaces.